docs/posts/index.md

```diff
@@ -0,0 +1,18 @@
+---
+aside: false
+---
+
```

Comment:
This file is the blog index, displaying a list of posts drawing from the metadata provided with frontmatter.

There's a filter to prevent listing itself and the example template.